### DISCUSSION TITLE
Paylaşılan doğum günü pastası

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#162] [Soru Cevap] [answered]
Bir doğum günü pastası toplam 100 kişi arasında paylaştırılmak isteniyor ancak kural şu:
1 inci pastanın %1' ini alıyor
2 inci kalan pastanın %2'sini alıyor
3. alan kalan pastanın %3'ünü alıyor
................................................. 100. kişi pastanın %100 ünü alıyor
Soru şu : En çok pastayı kaçıncı kişi alır?

**Comment by OmerB2** [comment, marked as answer]
```python
pasta = 100
encok = 0
kisi = 0
for i in range(1,101):
    alinan = (i*pasta)/100
    pasta = pasta-alinan
    if alinan>encok:
        encok = alinan
        kisi = i
print("En cok",kisi,"numarali kisi pasta almistir.")
```

**Reply by sonsuzus** [reply]
Bu koda göre yanlış Kişi alır sanki

**Reply by OmerB2** [reply]
hocam bu doğru mu?

**Reply by sonsuzus** [reply]
Şimdi olmuş.
